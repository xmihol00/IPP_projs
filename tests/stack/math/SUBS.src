.IPPcode21

DEFVAR GF@num1
DEFVAR GF@num2

PUSHS int@45
PUSHS int@10
SUBS
POPS GF@num1
WRITE GF@num1
WRITE string@\010

PUSHS float@0x1.f9p+5
PUSHS float@0x1.56p+6
SUBS
POPS GF@num2
WRITE GF@num2
WRITE string@\010

PUSHS GF@num1
INT2FLOATS
PUSHS GF@num2
SUBS
POPS GF@num2
WRITE GF@num2
WRITE string@\010

PUSHS int@45
PUSHS GF@num2
FLOAT2INTS
SUBS
POPS GF@num2
WRITE GF@num2
WRITE string@\010
