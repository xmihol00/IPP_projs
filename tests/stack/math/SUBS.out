35
-0x1.6600000000000p+4
0x1.cb00000000000p+5
-12
